interface-definitions/service_ids_suricata.xml.in

```diff
@@ -0,0 +1,127 @@
+                    <regex>[A-Z_]+</regex>
+                  </constraint>
+                </properties>
+                <children>
+                  <leafNode name="value">
+                    <properties>
+                      <help>Address group values</help>
+                      <valueHelp>
+                        <format>ipv4net</format>
+                        <description>IPv4 prefix(es) to group</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>ipv6net</format>
+                        <description>IPv6 prefix(es) to group</description>
+                      </valueHelp>
+                      <multi/>
+                    </properties>
+                  </leafNode>
+                </children>
+              </tagNode>
+              <tagNode name="port-group">
+                <properties>
+                  <help>Port group name</help>
+                  <completionHelp>
+                    <list>HTTP_PORTS SHELLCODE_PORTS ORACLE_PORTS SSH_PORTS DNP3_PORTS MODBUS_PORTS FILE_DATA_PORTS FTP_PORTS GENEVE_PORTS VXLAN_PORTS TEREDO_PORTS</list>
+                  </completionHelp>
+                  <constraint>
+                    <regex>[A-Z_]+</regex>
+                  </constraint>
+                </properties>
+                <children>
+                  <leafNode name="value">
+                    <properties>
+                      <help>Port group values</help>
+                      <multi/>
+                    </properties>
+                  </leafNode>
+                </children>
+              </tagNode>
+              <node name="output">
+                <properties>
+                  <help>Suricata log outputs</help>
+                </properties>
+                <children>
+                  <node name="eve">
+                    <properties>
+                      <help>Extensible Event Format (nicknamed EVE) event log in JSON format</help>
+                    </properties>
+                    <children>
+                      <leafNode name="filetype">
+                        <properties>
+                          <help>EVE logging destination</help>
+                          <completionHelp>
+                            <list>regular syslog</list>
+                          </completionHelp>
+                          <valueHelp>
+                            <format>regular</format>
+                            <description>Log to filename</description>
+                          </valueHelp>
+                          <valueHelp>
+                            <format>syslog</format>
+                            <description>Log to syslog</description>
+                          </valueHelp>
+                          <constraint>
+                            <regex>(regular|syslog)</regex>
+                          </constraint>
+                        </properties>
+                      </leafNode>
```

Comment:
Do we need the `defaultValue` here? 
For example https://github.com/vyos/vyos-1x/blob/f97a3254c9e958f76b3ff3dfec7e9297cf48575c/interface-definitions/container.xml.in#L309